integration/bdpl_update_test.go

```diff
@@ -290,7 +290,7 @@ var _ = Describe("BDPL updates", func() {
 			It("should update the dns pods", func() {
 				deployment, err := env.CollectDeployment(env.Namespace, "bosh-dns", 1)
 				Expect(err).NotTo(HaveOccurred())
-				Expect(deployment.Spec.Template.GetAnnotations()).To(HaveKey(quarksrestart.RestartKey))
+				Expect(deployment.Spec.Template.GetAnnotations()).To(HaveKey(quarksrestart.AnnotationRestartOnUpdate))
```

Comment:
Does this change the meaning of the test? Like from "has restarted" to "should restart"?

Reply:
Nice catch. This was a mistake.